contracts/RootChain.sol

```diff
@@ -239,28 +238,29 @@ contract RootChain {
      * @param _txBytes The challenging transaction in bytes RLP form.
      * @param _proof Proof of inclusion for the transaction used to challenge.
      * @param _sigs Signatures for the transaction used to challenge.
-     * @param _confirmationSig The confirmation signature for the transaction used to challenge.
      */
     function challengeExit(
         uint256 _cUtxoPos,
         uint256 _eUtxoIndex,
         bytes _txBytes,
         bytes _proof,
-        bytes _sigs,
-        bytes _confirmationSig
+        bytes _sigs
```

Comment:
Seems easier to have the challenger provide a single signature and check that `ECRecovery.recover(keccak256(_txBytes), _sig) == owner)`. Gets rid of a lot of extra logic about the `oindex` and removes the need for `Validate.checkSigs` entirely.

Reply:
This would require changes in client. Lets punt this to more-vp stage.